Add migration error messages to V2 (#5480)
pydantic/pydantic: docs/usage/errors.md

TODO

## Import error {#import-error}

> TODO

Author: Good enough?
```suggestion
This error is raised when you try to import an object that was available on V1, but it's not anymore on V2.
```

> This error is raised when you try to import an object that was available on V1, but it's not anymore on V2.

Reviewer: ```suggestion
This error is raised when you try to import an object that was available in V1 but has been removed in V2.
```
